templates/apps/sumo-aws-apps/sumologic-app-utils/src/awsresource.py

```diff
@@ -277,7 +277,7 @@ def enable_aws_service_access(self,service_principal: str):
         :param service_principal: AWS Service Principal
         :return: None
         """
-        logger.info(f"Enabling AWS Service Access for: {service_principal}")
+        #logger.info(f"Enabling AWS Service Access for: {service_principal}")
```

Comment:
service_principal does not contain any sensitive info I think we can put this in debug log level